tests/Issues/Issue238/MissingVariablesTest.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace Youshido\Tests\Issues\Issue238;
+
+use PHPUnit\Framework\TestCase;
+use Youshido\GraphQL\Execution\Processor;
+use Youshido\Tests\StarWars\Schema\StarWarsSchema;
+
+class MissingVariablesTest extends TestCase
+{
+
+    /**
+     * @see https://github.com/youshido-php/GraphQL/issues/238
+     */
+    public function testProperExceptionOnMissingVariables()
+    {
+        $payload = <<<GQL
+query GetHero {
+    hero(episode:\$episode) {
+        name
+    }
+}
+GQL;
+
+        $schema = new StarWarsSchema();
+        $processor = new Processor($schema);
+
+        $processor->processPayload($payload);
+        $response = $processor->getResponseData();
+
+        $this->assertEquals([
+            'errors' => [
+                [
+                    'message' => 'Variable "episode" does not exists',
```

Comment:
@leoloso this is the error you expect, right?

Reply:
I think the more important error is that the variable is not defined in the operation name, which is mandatory in GraphQL. Hence the error should be something like `Variable "episode" has not been defined in the operation name`